vJson/endPoint/deleteItem.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require_once 'config.php';
+require_once 'validation.php';
+
+function deleteItem(array $idItem): array
+{
+    $todoList = json_decode(file_get_contents(TODO_FILE), true);
+
+    foreach ($todoList as $key => &$todoItem) {
+        if ($todoItem['id'] == $idItem['id']) {
+            unset($todoList[$key]);
+        }
+
+        if ($todoItem['id'] > $idItem['id']) {
+            $todoItem['id']--;

```

Comment:
no sense in reducing id. In further you mostly will work with mysql database. reducing id there will bring you problems